### ISSUE TITLE
ReferenceError: node is not defined

### ISSUE THREAD

**Issue by KMax** [kmi/node-red-contrib-swagger#2]
I installed the plugin to Node-RED v0.12.2, but once I open the UI I get the following exception and node-red stops:

```
Welcome to Node-RED
===================

20 Jan 18:10:32 - [info] Node-RED version: v0.12.2
20 Jan 18:10:32 - [info] Node.js  version: v5.4.1
20 Jan 18:10:32 - [info] Loading palette nodes
20 Jan 18:10:32 - [warn] ------------------------------------------
20 Jan 18:10:32 - [warn] [rpi-gpio] Info : Ignoring Raspberry Pi specific node
20 Jan 18:10:32 - [warn] [serialport] Error: Cannot find module '/usr/lib/node_modules/node-red/node_modules/node-red-node-serialport/node_modules/serialport/build/serialport/v1.7.4/Release/node-v47-linux-x64/serialport.node'
20 Jan 18:10:32 - [warn] ------------------------------------------
20 Jan 18:10:32 - [info] Settings file  : /home/user/.node-red/settings.js
20 Jan 18:10:32 - [info] User directory : /home/user/.node-red
20 Jan 18:10:32 - [info] Flows file : /home/user/.node-red/flows_localhost.localdomain.json
20 Jan 18:10:32 - [info] Server now running at http://127.0.0.1:1880/
20 Jan 18:10:32 - [info] Starting flows
20 Jan 18:10:32 - [info] Started flows
20 Jan 18:10:40 - [red] Uncaught Exception:
20 Jan 18:10:40 - ReferenceError: node is not defined
    at ReadFileContext.callback (/usr/lib/node_modules/node-red/node_modules/node-red-contrib-swagger/swagger/node-red-contrib-swagger.js:300:21)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:324:13)

```

**Comment by m67hoff** [comment]
I have the same issue - 

(This project was last updated  more than a year ago . I consider it as probably dead)

**Comment by cpedrinaci** [comment]
I'll look into it

**Comment by MattiasMM** [comment]
Any update on this topic?
thanks!

**Comment by cpedrinaci** [comment]
Should be fixed in v0.3.2
